internal/storage/sql.go

```diff
@@ -124,38 +124,78 @@ var (
 
 	// insert cluster
 	InsertCluster = `
-		INSERT INTO clusters(uuid, name, description, topology, type, pool, create_time)
+		INSERT INTO clusters_new(uuid, name, description, topology, type, pool, create_time)
 		VALUES(?, ?, ?, ?, ?, "", datetime('now','localtime'))
 	`
-	// check new cluster column
-	GetTypeFiled = `SELECT COUNT(*) FROM pragma_table_info('clusters') WHERE name = 'type'`
+	// check if view exists
+	CheckMigrationNeeded = `SELECT COUNT(*) FROM sqlite_master WHERE type='view' AND name = 'clusters'`
```

Comment:
`isViewExist` may be better.  `CheckMigrationNeeded` is more suitable for the function below.

Reply:
fixed